src/quo2/components/selectors/react_selector/view.cljs

```diff
@@ -0,0 +1,29 @@
+(ns quo2.components.selectors.react-selector.view
+  (:require [quo2.components.markdown.text :as text]
+            [quo2.components.selectors.reaction-resource :as reaction.resource]
+            [quo2.components.selectors.react-selector.style :as style]
+            [quo2.theme :as theme]
+            [react-native.core :as rn]))
+
+(defn- view-internal
+  "Add your emoji as a param here"
```

Comment:
imo this doc string doesn't add any value here. If you could either update it to be a doc of the api or consider removing it 👍

Reply:
I guess if we end up using `malli`, the doc will be worthless so might as well remove it.